content/backmatter.tex

```diff
@@ -76,30 +89,30 @@ \section*{Incorporating OpenSHMEM into Programs}\label{sec:writing_programs}
 
 
 \chapter{Compiling and Running Programs}\label{sec:compiling}
-As of this writing, the \openshmem specification is silent regarding how
-\openshmem programs are compiled, linked and run. This section shows some
+The specification does not specify how
+\openshmem programs are compiled, linked, and run. This section shows some
 examples of how wrapper programs are utilized in the \openshmem Reference
 Implementation to compile and launch programs.
 
 \section{Compilation}
 \subsection*{Programs written in \Cstd}
 
-The \openshmem Reference Implementation provides a wrapper program named
-\textbf{oshcc}, to aid in the compilation of \Cstd programs, the wrapper
-could be called as follows:
+The \openshmem Reference Implementation provides a wrapper program, named
+\textbf{oshcc}, to aid in the compilation of \Cstd programs.
+The wrapper may be called as follows:
 
 \begin{lstlisting}[language=bash]
 oshcc <compiler options> -o myprogram myprogram.c
 \end{lstlisting}
 Where the $\langle\mbox{compiler options}\rangle$ are options understood by the
-underlying \Cstd compiler.
+underlying \Cstd compiler called by \textbf{oshcc}.
 
 
 \subsection*{Programs written in \Cpp}
 
-The  \openshmem Reference Implementation provides a wrapper program named
-\textbf{oshCC}, to aid in the compilation of \Cpp programs, the wrapper could
-be called as follows:
+The \openshmem Reference Implementation provides a wrapper program, named
+\textbf{oshCC}, to aid in the compilation of \Cpp programs.
```

Comment:
IIRC, this wrapper was renamed for compatibility with case-insensitive filesystems.